### DISCUSSION TITLE
Urgent Help Needed: Supporting Education and Family Through Hard Work

### DISCUSSION THREAD

**Discussion by MiantsaFanirina** [community/community#145286] [Programming Help]
### Body

Hi everyone, 

I’m Rakotondrafara Miantsa Fanirina, a 21-year-old student from Antananarivo, Madagascar. I’m currently in my 2nd year studying Computer Technology at ESTI (École Supérieure des Technologies de l'Information). My goal is to complete my degree and earn my license in October 2025. 

Unfortunately, due to the economic situation in my country and increasing inflation, my parents can no longer afford my scholarship. My father, a high school sports teacher, and my mother, who sells basic necessities (PPN), are struggling to cover even the essentials like bills and food. 

Why I’m Posting 
My scholarship fee of $750 is due tomorrow, and I don’t want to burden my parents anymore. I’m looking for someone to help me cover this cost in exchange for my skills. 

I’d like to request $1000: 
- $750 for my education 
- $250 to help my parents 

This is not a charity request. I am offering to work and earn this money through my skills and dedication. 

My Skills 
- I’m a full-stack developer with strong expertise in JavaScript and TypeScript. Here’s what I bring to the table: 
- Proficient in: ReactJS, NextJS, NestJS, and some experience with Angular 
- Design tools: Tailwind CSS, Figma, and Adobe XD 
- Coding practices: Good design patterns and clean, maintainable code 
- Experience: Building responsive, user-friendly web applications 
I’m not a beginner, I take pride in delivering high-quality work and learning quickly to adapt to project needs. 

What You Get 
- A hardworking developer for less than you’d pay a regular worker 
- Commitment to best practices and delivering results 
- Flexibility and dedication to meet your needs 

My Availability 
I study half a day, so I can work from 1 PM to 9 PM (8 hours/day) Madagascar time. 

Why I Need Your Help Now 
Finding a job as a student is challenging, especially in my country, where opportunities are limited. Freelancing takes time to establish, and I don’t have an existing portfolio to attract immediate clients. However, I have 24/7 internet access and am ready to start immediately. 

A Final Note 
I understand trust is hard to build, especially when it involves money. I promise to give my absolute best to earn your support and will not let you down. 

If this post is not appropriate for this community, I truly apologize and appreciate your understanding. 

If you have any job opportunities or know someone who does, please reach out to me at [RakotondrafaraMiantsaFanirina@gmail.com](mailto:RakotondrafaraMiantsaFanirina@gmail.com). Even small tasks will help me take a step closer to securing my education and supporting my family. 

Thank you so much for reading. Any help, advice, or job offer would mean the world to me. 🙏

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by denis23x** [comment]
Hi @MiantsaFanirina 

It's sad to hear your story, I hope everything will be fine for you soon 🙏

Unfortunately I can't offer you a job, but I can advise you to look at sites like https://www.upwork.com and its alternatives. It is quite possible to find a side income.

**Comment by queenofcorgis** [comment]
Hi @MiantsaFanirina , thank you for posting and sharing your story. 

However, we currently do not allow job seeking on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
